### DISCUSSION TITLE
Curios about the setup of this website unsigned.io

### DISCUSSION THREAD

**Discussion by izdrail** [markqvist/Reticulum#274] [Help & Questions]
Like the setup complete ?
Do you use raspberry ?

**Comment by faragher** [comment]
Do you mean you wish to host it yourself? It looks like it's designed to be served directly, so you'd need to set up a web server and upload the files to an accessible directory.

If you want to use a Pi, it's absolutely possible! You're looking for a "LAMP stack" and can find more information here: https://pimylifeup.com/raspberry-pi-apache/

**Comment by izdrail** [comment]
I was curios about the solar power part 
what ind of setup your using

**Comment by markqvist** [comment]
The site is running on a custom build system that generates static html pages from a set of markdown sources. It is basically the same system, but with extra extensions and functionality, that runs the Reticulum website. You can find the source for that on my github profile.

Hardware-wise it is running on a low-power SBC, and the solar power system is a bit too detailed to get into here ;) Suffice to say that the ~8 watts that the webserver requires is a *very* low amount of the total power output of the system.
